### ISSUE TITLE
Take POA into account when scheduling hearings

### ISSUE THREAD

**Issue by nicholasholtz** [department-of-veterans-affairs/caseflow#9736]
There is not a clear outcome/purpose here, so more investigation needs to be done before anything could be done on this issue.

The Board has requested that they would like the ability to take into account POA when it comes to scheduling hearings. This may be based on an assertion by some VSOs that too many hearings are being booked for a single individual representative, or where the representative is unavailable.

There are open questions that should be considered in weighing this request, including, but probably not limited to
- What is the frequency that this becomes an issue?
- Is existing rescheduling functionality not an adequate workaround?
- What sort of added complexity would this require from the perspective of the algorithm used to prioritize hearings, and would it force hearings to be worked out of docket order? Is that legal?
- What use cases need to be addressed? Two that come to mind
  - Do not double book a POA
  - Do not book a POA in 2 separate ROs that are hours apart (same state)

**Comment by nicholer** [comment]
@chandracarney Might this have been addressed by #12418 (Add POA to the Schedule Veterans screen), and can therefore be closed?

**Comment by chandracarney** [comment]
I'm wondering if we can port some of these requirements into our virtual hearings work, and then close, such as 
```- Do not double book a POA
- Do not book a POA in 2 separate ROs that are hours apart (same state)```

We've circled around these questions for virtual queues as well so perhaps when addressing further scheduling or Daily Docket work we can introduce this constraint?
